src/System.Runtime/tests/System/EnumTests.cs

```diff
@@ -224,6 +224,7 @@ public static void Parse_Invalid(Type enumType, string value, bool ignoreCase, T
             parseMethod.Invoke(null, new object[] { enumType, value, ignoreCase, exceptionType });
         }
 
+#pragma warning disable xUnit1013 // Public method should be marked as test
         public static void Parse_Generic_Invalid<T>(Type enumType, string value, bool ignoreCase, Type exceptionType) where T : struct
```

Comment:
It shouldn't need to be public.  You can instead just change the GetMethod call above to using BingingFlags.Static | BindingFlags.NonPublic.